### ISSUE TITLE
Local Media Panel: Fix aXe violations to enable karma test

### ISSUE THREAD

**Issue by BrittanyIRL** [swissspidy/web-stories-wp#9954]
## Context 
While adding in aXe violation checks to the library panels, a few issues were flagged in the local media panel. 
![Screen Shot 2021-12-09 at 5.05.29 PM.png](https://images.zenhubusercontent.com/5e9dcd9a9ff5dfdaffcdf132/e2a3c63e-3a6a-4e4d-bb29-2fc57196d9bf)

Issues are with video captions and color contrast. I believe the color contrast to just be a false positive due to how they are styled, however the video captions is interesting here. In this screenshot, it's actually a gif. When we know the media type is a gif - would it serve to specify no captions to avoid this violation noise? 
I know that captions are in the works for 2022. 

## Acceptance Criteria: 
- Enable local media tab aXe test in `components/library/karma/libraryTabs` and have it pass
